builder/fixtures/buildpacks/unix/has-buildpack-config/bin/detect

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+# vim: set ft=sh
```

Comment:
Probably should remove this `vim: set ft=sh` bit

Reply:
Fixed